src/Resources/views/CRUD/display_enum.html.twig

```diff
@@ -10,9 +10,17 @@ file that was distributed with this source code.
 #}
 
 {%- apply spaceless %}
-    {%- if value is null -%}
-        &nbsp;
-    {%- else -%}
-        {{ value.name }}
-    {%- endif -%}
+    {% set value = use_value|default(false) ? value.value : value.name %}
+
+    {% if translation_domain|default(null) is null %}
+        {% set value = value %}
+    {% else %}
+        {% set value = value|trans({}, translation_domain) %}
+    {% endif %}
+
+    {% if safe|default(false) %}
```

Comment:
The safe option is not documented and not used

Reply:
You're right! Removed.